paasta_tools/autoscaling/autoscaling_cluster_lib.py

```diff
@@ -1279,19 +1281,28 @@ def get_mesos_utilization_error(
         return 0
 
     log.debug(repr(region_pool_utilization_dict))
-    free_pool_resources = region_pool_utilization_dict['free']
-    total_pool_resources = region_pool_utilization_dict['total']
     usage_percs = []
-    for free, total in zip(free_pool_resources, total_pool_resources):
+    for resource in ResourceInfo._fields:
+        free = getattr(region_pool_utilization_dict['free'], resource)
+        total = getattr(region_pool_utilization_dict['total'], resource)
+
         if math.isclose(total, 0):
             continue
+
         current_load = total - free
-        boosted_load = cluster_boost.get_boosted_load(region=region, pool=pool, current_load=current_load)
+
+        # We apply the boost only on the cpu resource.
+        if resource == 'cpus'and system_config.get_cluster_boost_enabled():
+            boosted_load = cluster_boost.get_boosted_load(region=region, pool=pool, current_load=current_load)
+        else:
+            boosted_load = current_load
 
         usage_percs.append(boosted_load / float(total))
 
     if len(usage_percs) == 0:  # If all resource totals are close to 0 for some reason
```

Comment:
Will this statement ever be true? Looks like it will only be 0 if we never run the for loop.

Reply:
It's been explicitely added in the commit 5c3a683b so I'm sure we can find more context about it. I agree to look into it but not part of this PR as I'm trying to make as few changes as possible to this lib